packages/slate-editor/src/modules/editor-v4-paragraphs/lib/normalizeRedundantParagraphAttributes.ts

```diff
@@ -1,10 +1,15 @@
 import { EditorCommands } from '@prezly/slate-commons';
+import type { ParagraphNode } from '@prezly/slate-types';
 import { isParagraphNode } from '@prezly/slate-types';
 import type { Editor, NodeEntry } from 'slate';
 
-import { createParagraph } from './createParagraph';
+const shape: { [P in keyof ParagraphNode]: true } = {
```

Comment:
nit. you can use Record here

Reply:
Thanks! 

Done in 48c535c09c114d0947cbe7ccbb6081e823989a09